src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Classes/WindowsSetting.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes
+{
+    /// <summary>
+    /// A windows setting
+    /// </summary>
+    internal class WindowsSetting : IWindowsSetting
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WindowsSetting"/> class.
+        /// </summary>
+        public WindowsSetting()
+        {
+            Area = string.Empty;
+            Name = string.Empty;
+            Command = string.Empty;
```

Comment:
Are some elements missing here: alt names, introducedversion, depreciatedversion, note ?

Reply:
Theses values can have `null` as value, so we need to set values for that